app/experimenter/experiments/views.py

```diff
@@ -233,3 +233,12 @@ def form_invalid(self, form):
 
 class ExperimentRapidView(TemplateView):
     template_name = "experiments/rapid.html"
+
+
+class ExperimentNimbusUIView(TemplateView):
+    template_name = "experiments/nimbus-ui.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context["debug"] = settings.NIMBUS_DEV
+        return context
```

Comment:
@jaredlockhart tests keep yelling at me that this isn't getting tested, but afaict it is [here](https://github.com/mozilla/experimenter/pull/3670/files#diff-d62dba79b6a4e33ab663cf5e648aee8fR826). What would I be missing?

Reply:
Oh if you look up you'll see that the test is failing becuase `experiments-nimbus` isn't a valid url name, looks like you forgot to rename it here:

https://github.com/mozilla/experimenter/pull/3670/files#diff-23077f4ae423069e9d89856456c09167R23-R25


The coverage is complaining those lines aren't covered becuase the test failed, so those lines never get hit.  Usually you wanna look at test failures first, then if they're all passing, then look at coverage.

Reply:
Ahhhh thank you. My bad!